testsuite/features/step_definitions/command_steps.rb

```diff
@@ -1444,18 +1444,22 @@
     end
 
     command = "spacecmd -u admin -p admin proxy_container_config -- -o #{file_path} -p 8022 " \
-              "#{get_target('proxy').full_hostname.sub('pxy', 'pod-pxy')} #{get_target('server').full_hostname} 2048 galaxy-noise@suse.de " \
+              "#{get_target('proxy').full_hostname} #{get_target('server').full_hostname} 2048 galaxy-noise@suse.de " \
               '/tmp/ca.crt /tmp/proxy.crt /tmp/proxy.key'
   else
-    # Doc: https://www.uyuni-project.org/uyuni-docs/en/uyuni/reference/spacecmd/proxy_container.html
-    command = 'echo spacewalk > cert_pass && spacecmd -u admin -p admin proxy_container_config_generate_cert' \
-              " -- -o #{file_path} -p 8022 #{get_target('proxy').full_hostname.sub('pxy', 'pod-pxy')} #{get_target('server').full_hostname}" \
-              ' 2048 galaxy-noise@suse.de --ca-pass cert_pass' \
-              ' && rm cert_pass'
+
+    command = "echo spacewalk > ca_pass && spacecmd --nossl -u admin -p admin proxy_container_config_generate_cert -- -o #{file_path} #{get_target('proxy').full_hostname} #{get_target('server').full_hostname} 2048 galaxy-noise@suse.de --ca-pass ca_pass"
```

Comment:
The new command looks slightly different. 
- The port `8022` is missing
  The docs state
 >   -p, --ssh-port SSH port the proxy listens one. Default: 22
-  `rm cert_pass` is missing. Is this intentional?

Reply:
It works without specifying the port 👀 And I also don't have it in my notes when I has been testing it.

Reply:
I will for now keep it without specifying the port.
I will review this with @cbosdo , probably we need to change the doc.

Reply:
> The new command looks slightly different.
> 
>     * The port `8022` is missing (`-p 8022`)
>       The [docs](https://www.uyuni-project.org/uyuni-docs/en/uyuni/reference/spacecmd/proxy_container.html) state
> 
> 
> > -p, --ssh-port SSH port the proxy listens one. Default: 22

Where do you see the `22` default? `spacecmd` help clearly states it's `8022`. Indeed: the port is usually not needed.

Reply:
The docs say, it defaults to port 22. That is why I am asking.

Reply:
Could you provide a link?

Reply:
https://www.uyuni-project.org/uyuni-docs/en/uyuni/reference/spacecmd/proxy_container.html 

```bash
usage: proxy_container_config_generate_cert PROXY_FQDN PARENT_FQDN MAX_CACHE EMAIL

parameters:
  PROXY_FQDN  the fully qualified domain name of the proxy to create.
  PARENT_FQDN the fully qualified domain name of the server or another proxy to connect to.
  MAX_CACHE   the maximum cache size in MB. 60% of the storage is a good value.
  EMAIL       the email of the proxy administrator

options:
  -o, --output Path where to create the generated configuration. Default: 'config.zip'
  -p, --ssh-port SSH port the proxy listens one. Default: 22
(...)
```